README.md

```diff
@@ -76,19 +67,21 @@ chmod +x run-unit-tests.sh
 ./run-unit-tests.sh
 ```
 
-## Build
-
+### 3. Deploy
 ```bash
-cd $MAIN_DIRECTORY/deployment
-chmod +x build-s3-dist.sh
-./build-s3-dist.sh $BUCKET_PREFIX $SOLUTION_NAME $VERSION
+cd $MAIN_DIRECTORY/source/constructs
+npm run clean:install
+npx cdk bootstrap --profile <PROFILE_NAME> --region <AWS_REGION>
```

Comment:
Should this be:
```
overrideWarningsEnabled=false npx cdk bootstrap --profile <PROFILE_NAME> --region <AWS_REGION>
```

Without it I am getting a long hang followed by an OOM.

Reply:
addressed